### ISSUE TITLE
Quill forgets tokens

### ISSUE THREAD

**Issue by sebsel** [aaronpk/Quill#73]
This happened twice to me already: Quill seems to forget the access token it stored for my site after a while. The settings show this:

![quill-no-token](https://cloud.githubusercontent.com/assets/16517999/26520204/71bee8e2-42ce-11e7-9b03-d9bbe30de5f7.jpg)

Previously, it showed the access token, but the field is blank now and from the errors my site gives I can deduce that Quill also sends an empty Bearer when posting.

Also note that the scope changed from `create update media` (which I gave it) to just `post`, but I think that's just because it defaults to that if it can't look it up from the token?

**Comment by aaronpk** [comment]
Strange that I haven't seen this happen before. I think you're hitting an edge case that is marked with a nice "todo" in the code: https://github.com/aaronpk/Quill/commit/91307d4a362a8fab0022aba19ab74bec9cb46f8f

Did you notice that this happened after you changed your auth/token/micropub endpoint on your site? I think that's the code path this is following.

**Comment by sebsel** [comment]
Yes, it began since I changed my AuthEndpoint to seblog.nl/auth, but my TokenEndpoint is still `https://tokens.indieauth.com/token`.

When I'm just signed in, the access token is there, but when I come back after a while it's gone. I don't get prompted to login again, however, it feels like the same session and I don't hit seblog.nl/auth. Does the code you pointed to still run in that case?

Now I tried to login after pressing the reset button. Let's see if that solves it. (It should update my auth-endpoint then, right?)
